grapple/helpers.py

```diff
@@ -51,11 +51,17 @@ def register_query_field(
 
     def inner(cls):
         field_type = lambda: registry.models[cls]
-        field_query_params = query_params or {
```

Comment:
@ruisaraiva19 changed the other two decorators to add the default slug/token params only for page subclasses

Reply:
nice!